fuse_constraints/src/marginalize_variables.cpp

```diff
@@ -321,20 +327,24 @@ LinearTerm linearize(
                              "during the jacobian computation.");
   }
 
-  // Update the jacobians with the local parameterizations.
+  // Update the jacobians with the local parameterizations. And clear the jacobians is the variable is held constant.
```

Comment:
This sentence is hard to grasp the meaning. Consider changing to another way?

Reply:
```
  // Update the jacobians with the local parameterizations. This potentially changes the size of the Jacobian block.
  // The classic example is a quaternion parameter, which has 4 components but only 3 degrees of freedom. The Jacobian
  // will be transformed from 4 columns to 3 columns after the local parameterization is applied.
  // We also check for variables that have been marked as constants. Since these variables cannot change value, their
  // derivatives/Jacobians should be zero.
```
Is that better?

Reply:
Very clear.